app/helpers/pulitzer/main_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Pulitzer
+  module MainHelper
+    # Will this clash with helper methods like devise, cancan?
+    def authorized?
```

Comment:
should't we change this method name in case it clashes with another one from devise or something similar?